.travis.yml

```
@@ -16,14 +16,41 @@ jobs:
   - python: 3.9-dev
   - <<: *latest_py3
     env: TOXENV=docs
+    #PPC64LE
+  - arch: ppc64le
```

It's best to add job names so that it's visible in the UI as @jaraco asked earlier (https://github.com/pypa/setuptools/pull/2379#pullrequestreview-482091500).
Please add `name: ` to all the jobs you've added, similar to:
```suggestion
  - arch: ppc64le
    name: pypy3 @ ppc64le
```

---

Ah, scratch that, I missed a comment about the second column in the UI.